gitlab-ci/pack.ps1

```diff
@@ -0,0 +1,125 @@
+Param(
```

Comment:
Emmm, is this the same file as win/pack.ps1?

Reply:
Not exactly. This file only prepares the folder structure. It does has many dupe codes of determining install prefix. Maybe we should bring that part as some utility functions?

Reply:
Then why do we need this, could you clarify the use case?

BTW I don't like the idea of having function in script, as it causes non-linear control flow.
For reusing code, switches/subscripts are usually better choice.

Reply:
like the filename suggests, this will be used in ci and it prepares desired folder structures for nuget package. win/pack.ps1 is similar with win/setup.ps1, which is target to be used locally.

and i can't agree the idea of not having function in script. The readability will be too bad. At least for powershell, I barely see scripts without any functions.